sql/migrations/20221027213705_update_coverage_field_precision.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE test
+  ALTER COLUMN coverage TYPE NUMERIC(5,2);
```

Comment:
https://github.com/digitalocean/gocop/blob/v0.1.2/action/store.go#L78

I believe `NUMERIC(4,2)` precision was intentional since the coverage has always been divided by 100.